README.md

```diff
@@ -71,65 +72,96 @@ Provides the following HTTP methods for `/load-test` endpoint:
 
  If you prefer to use Postman you can also import [openapi.json](openapi.json) file into Postman to create a new collection.
 
-### Kangal controller
-The general name for several Kubernetes controllers created to manage all the aspects of the performance testing process.
- - LoadTest controller  
- - Backend jobs controller
- 
-## To start using Kangal
-To run Kangal in your Kubernetes cluster follow [docs](docs/README.md#how-do-i-use-kangal)
-
-Also check out our [User Flow](docs/Kangal-user-flow.md) guide to start creating load tests with Kangal.
-
-More detailed information can be found in [docs folder](docs/)
-
-## To start developing Kangal
-To start developing Kangal you need a local Kubernetes environment, e.g. [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/)
-or [docker desktop](https://rominirani.com/tutorial-getting-started-with-kubernetes-with-docker-on-mac-7f58467203fd). 
-> Note: Depending on load generator type, load test environments created by Kangal may require a lot of resources. Make sure you increased your limits for local Kubernetes cluster. 
-> Read more about implemented load generators [here](docs/README.md#load-generator-types-aka-backends). 
-
-1. Clone the repo locally
-    ```bash
-    git clone https://github.com/hellofresh/kangal.git
-    ```
-
-2. Create required Kubernetes resource LoadTest CRD in your cluster
-    ```bash
-    kubectl apply -f charts/kangal/crd.yaml
-    ```
-    or just use:
-    ```bash
-    make appply-crd
-    ```
-    
-3. Download the dependencies
-    ```bash
-    go mod vendor
-    ```
-
-4. Build Kangal binary
-   ```bash
-   make build
-    ```
-    
-5. Set the environment variables
-    ``` bash
-    export WEB_HTTP_PORT=8080                    # API port for Kangal Proxy
-    export AWS_BUCKET_NAME=YOUR_BUCKET_NAME      # name of the bucket for saving reports
-    export AWS_ENDPOINT_URL=YOUR_BUCKET_ENDPOINT # storage connection parameter
-    export AWS_DEFAULT_REGION=YOUR_AWS_REGION    # storage connection parameter
-    ```
-
-6. Run both Kangal proxy and controller
-    ```bash
-    ./kangal controller --kubeconfig=$KUBECONFIG 
-    ./kangal proxy --kubeconfig=$KUBECONFIG
-    ```
+### Kangal Controller
+The component is responsible for managing all the aspects of the performance testing process.
 
-## Contributing
+## Quickstart guide
+This tutorial will guide through Kangal installation process and usage.
+
+### Installing using helm
+First, let's create the Custom Resource Definition by running:
+
+```shell
+$ kubectl apply -f https://raw.githubusercontent.com/hellofresh/kangal/master/charts/kangal/crd.yaml
+```
+
+Now, install the chart using following commands:
+
+```shell
+$ helm repo add kangal https://hellofresh.github.io/kangal
+$ helm install --set environment=dev kangal kangal/kangal
+```
```

Comment:
That all is already here:
https://github.com/hellofresh/kangal/blob/master/charts/kangal/README.md#installing-the-chart

I am wondering of this duplicates, can we just point directly to this Readme instead?

Reply:
Pointing makes things harder to find, it's good to have quickstart instructions right on the first page.